src/main/scala/edg_ide/runner/DesignTopRunConfigurationType.scala

```diff
@@ -80,37 +92,66 @@ class DesignTopRunConfiguration(project: Project, factory: ConfigurationFactory,
 
   val kFieldDesignName = "DESIGN_NAME"
   val kFieldNetlistName = "NETLIST_NAME"
+  val kFieldToggle = "TOGGLE_NAME"
+
+  // Allows persistence of run configuration
   override def readExternal(element: Element): Unit = {
     super.readExternal(element)
     options.designName = JDOMExternalizerUtil.readField(element, kFieldDesignName)
     options.netlistFile = JDOMExternalizerUtil.readField(element, kFieldNetlistName)
+    Toggle.toEnum(JDOMExternalizerUtil.readField(element, kFieldToggle)).foreach(options.toggle = _)
   }
+
   override def writeExternal(element: Element): Unit = {
     super.writeExternal(element)
     JDOMExternalizerUtil.writeField(element, kFieldDesignName, options.designName)
     JDOMExternalizerUtil.writeField(element, kFieldNetlistName, options.netlistFile)
+    JDOMExternalizerUtil.writeField(element, kFieldToggle, options.toggle.toString)
   }
 }
 
-
 class DesignTopSettingsEditor(project: Project) extends SettingsEditor[DesignTopRunConfiguration] {
   protected val designName = new JTextField()
   protected val netlistFile = new JTextField()  // no browse button b/c FileChooser can't create new files
 
+  protected val toggleRefdes = new JBRadioButton("Refdes")
```

Comment:
Is there anywhere you can add a descriptive label for this, so a user can know what the radio buttons do? eg, "Netlist refdes value"

Is there also a way to make it somewhat visually consistent with the style of the textboxes, where they have the side-by-side descriptive labels?